src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -993,8 +993,9 @@ trait Definitions extends api.StandardDefinitions {
             // must filter out "universal" members (getClass is deferred for some reason)
             val deferredMembers =
               tpSym.info.membersBasedOnFlags(excludedFlags = BridgeAndPrivateFlags, requiredFlags = METHOD)
-              .toList
+              .iterator
```

Comment:
Not directly related to the change, more of a BTW: of `deferredMembers` only the first element is of interest, and only if the scope has size one. We can tackle that later, of course.